src/components/PageSections/Footer/_Footer.scss

```diff
@@ -31,56 +31,94 @@
       color: $coa-color-primary;
     }
   }
-  @include tablet-and-desktop {
-    padding-left: 10rem;
-    padding-right: 10rem;
+  @include desktop {
+    padding-left: 10rem; // We don't have a var for spacing this big
+    padding-right: 10rem; // We don't have a var for spacing this big
   }
-  @include mobile {
+  @include mobile-and-tablet {
     text-align: center;
   }
 }
 
 .coa-Footer__city-seal-wrapper {
   text-align: center;
-  // padding: $coa-space-level5 0 $coa-space-level3;
-  // @include media($coa-medium-screen) {
-  //   padding: $coa-space-level6 0 $coa-space-level3;
-  // }
-  svg {
-    height: 180px !important;
-    width: 180px !important;
+  margin-top: $coa-space-level7;
+  margin-bottom: $coa-space-level4;
+  img {
+    width: $coa-footer-seal-size;
+    height: $coa-footer-seal-size;
+  }
+
+  @include desktop {
+    img {
+      // not sure how well this calc holds up if we change the seal size,
+      // but it looks good for now
+      margin-bottom: calc(#{$coa-footer-seal-size} / -1.8);
+    }
+
+    border-bottom: 1px solid $coa-color-footer-lightest;
   }
 }
 
 .coa-Footer__work-in-progress {
-  text-align: center;
   font-size: $coa-font-size-small;
-  line-height: $coa-line-height-small;
-  font-weight: $coa-font-bold;
-  white-space: pre-line;
+  line-height: $coa-line-height-xsmall;
+  margin-bottom: $coa-space-level4;
 
-  @include media($coa-medium-screen) {
-    font-size: $coa-font-size-medium;
-    line-height: $coa-line-height-medium;
-    font-weight: $coa-font-regular;
+  a {
+    text-decoration: underline;
+    color: $coa-color-footer-babyblue;
+    &:visited {
+      color: $coa-color-footer-babyblue;
+    }
+  }
+
+  @include desktop {
+    margin-left: 10rem; // We don't have a var for spacing this big
+    margin-top: $coa-space-level4;
+    max-width: $coa-footer-wip-max-width;
+  }
+
+  @include mobile {
+    max-width: $coa-footer-wip-max-width;
   }
 
-  max-width: 660px;
-  margin: auto;
+  @include mobile-and-tablet {
+    text-align: center;
+    margin-left: auto;
+    margin-right: auto;
+  }
+}
 
+.coa-Footer__social-links {
   a {
     color: $coa-color-white;
     &:visited {
       color: $coa-color-white;
     }
-    svg {
-      margin-left: $coa-space-level1;
-      vertical-align: middle;
-      path {
-        fill: $coa-color-white;
-      }
+    &:hover {
+      color: $coa-color-blue-light;
+    }
+    padding-left: $coa-space-level4;
+    padding-right: $coa-space-level4;
+
+    @include tablet {
+      padding-left: $coa-space-level7;
+      padding-right: $coa-space-level7;
     }
   }
+  padding-top: $coa-space-level4;
+  padding-bottom: $coa-space-level4;
+  text-align: center;
+  margin-top: $coa-space-level4;
+  border-top: 1px solid $coa-footer-other-gray;
+
+  @include desktop {
+    border-top: none;
+    text-align: right;
+    margin-top: -8rem;
```

Comment:
I know this is an antipattern but it looks really good and I didn't feel like fighting flexbox on it.